backend/app/controllers/api/v1/projects/maps_controller.rb

```diff
@@ -0,0 +1,19 @@
+module API
+  module V1
+    module Projects
+      class MapsController < BaseController
+        def show
+          projects = Project.select(:id, :geometry, :latitude, :longitude)
```

Comment:
I think we don't need geometry here - just lat / long. Leaving out geometry will make this response significantly smaller I think. But I would add the "trusted" flag here, in case we need to differentiate display of markers.